### ISSUE TITLE
[Bug]: react/jsx-no-target-blank doesn't seem to work?

### ISSUE THREAD

**Issue by SkylerRogers** [jsx-eslint/eslint-plugin-react#3620]
### Is there an existing issue for this?

- [X] I have searched the existing issues and my issue is unique
- [X] My issue appears in the command-line and not only in the text editor

### Description Overview

It seems as if no matter how I add the `react/jsx-no-target-blank` rule to my eslintrc, it doesn't seem to enforce the rule.

If you add `<a href="/" target="_blank">test</a>` to your jsx, the linter is not enforcing `rel="noopener"` and/or `rel="noreferrer"`

### Expected Behavior

If you add `<a href="/" target="_blank">test</a>` to your jsx, the linter should enforce `rel="noopener"` and/or `rel="noreferrer"`

### eslint-plugin-react version

^7.31.11

### eslint version

^8.23.1

### node version

16.19.1

**Comment by ljharb** [comment]
This plugin is on v7.33.2. Have you tried updating?

**Comment by SkylerRogers** [comment]
Yeah admittedly we're a bit dated on all our dependencies for this app. Let me work on upgrading and get back with you. Thanks for the quick reply!

**Comment by SkylerRogers** [comment]
Upgrading and tweaking the rule config (e.g. adding  `warnOnSpreadAttributes: true`) appeared to fix the issue.

One other small note is that when I was first testing the rule I added something like this to my jsx `<a href="/" target="_blank">test</a>` and eslint wasn't detecting an issue. However, when I changed to proper URL (e.g. ` <a href="https://google.com/" target="_blank">test</a>`), the linter rule began to work again.

**Comment by ljharb** [comment]
`/` is a proper URL - it's the root of the site. Generally you want all URLs for the same site to be root-relative (starts with `/`)

**Comment by SkylerRogers** [comment]
Sure - well there seems to be a bug with relative path urls then..

**Comment by ljharb** [comment]
Relative links are on your own site, so you don't *need* noopener/noreferrer - because you can't distrust yourself.

**Comment by burtek** [comment]
@ljharb seems like this can be closed then?
